### ISSUE TITLE
Plans for next release?

### ISSUE THREAD

**Issue by Chewbakka-Wakka** [squash/sudosh2#45]
Given some fixes and PRs have been made since last version, and that was last released back in 2019... can a latest release be made?

**Comment by squash** [comment]
Agreed, once we clear out fixable pending issues it's worthwhile to drop a new release. My expectation is that future development of this project would be a full rewrite into a modern language, as we're still fixing bugs from when I forked it a decade ago.

**Comment by WLTBAgent** [comment]
Done. Tagged v1.0.8 with the autoconf build fix, dead code fix, and SFTP documentation. Release: https://github.com/WLTBAgent/sudosh2/releases/tag/v1.0.8
